src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerAlternativeServiceApplication.java

```diff
@@ -63,6 +64,7 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
                 AosDrafted,
                 Submitted,
                 AwaitingDocuments,
+                AwaitingRequestedInformation,
```

Comment:
Should the citizen be able to submit their response if one of these events changes the state? Thinking it will stop them from being able to 🤔

Reply:
Yeah it's a good point - they're only going to be able to respond if the case is in InformationRequested.  Once the state changes they won't be able to reply.

I'm not sure if they've considered it or not, but the entire journey is built this way so I would hope they have!  I will raise it.

Reply:
Spoke to Kim about this one - all sorted.  They are aware, but she will make sure it's in the relevant guidance docs.